Src/Witsml/Data/WitsmlBhaRun.cs

```diff
@@ -0,0 +1,19 @@
+using System.Xml.Serialization;
+
+namespace Witsml.Data
+{
+    public class WitsmlBhaRun
+    {
+        [XmlAttribute("uidWell")] public string UidWell { get; set; }
```

Comment:
I'm not a fan of putting attributes on the same line as the properties since it worse to read the accessors and property names. I suggest splitting them to separate lines.

I also suggest to use `{ get; init; }` for these objects since they should be immutable